sdk-core/build.gradle.kts

```diff
@@ -74,3 +74,31 @@ dependencies {
     testImplementation(Dependencies.Java.Test.okHttpMockWebServer)
     testImplementation(Dependencies.Java.Test.jsonAssert)
 }
+
+val templatesPath = "${projectDir}/src/main/resources/templates"
+val configPath = "${projectDir}/src/main/java/care/data4life/sdk/config"
```

Comment:
this should go to `src-gen` folder like in the FHIR SDK KMP to avoid accidential changes

Reply:
done